pkg/planner/util/handle_cols.go

```diff
@@ -77,11 +77,10 @@ type CommonHandleCols struct {
 	tblInfo *model.TableInfo
 	idxInfo *model.IndexInfo
 	columns []*expression.Column
-	sc      *stmtctx.StatementContext
```

Comment:
Could you also add some comments here to notify that this structure must be cross-session thread safe?

Reply:
To at least notify others the trick here via comments... and avoid them putting another non-thread-safe structures into it again.

Reply:
updated~